noxfile.py

```diff
@@ -0,0 +1,169 @@
+import os
+from contextlib import contextmanager
+from inspect import cleandoc
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import nox
+from pyodbc import connect
+
+PROJECT_ROOT = Path(__file__).parent
+
+# default actions to be run if nothing is explicitly specified with the -s option
+nox.options.sessions = ["verify"]
+
+
+class Settings:
+    ITDE = PROJECT_ROOT / ".." / "integration-test-docker-environment"
+    ODBC_DRIVER = PROJECT_ROOT / "driver" / "libexaodbc-uo2214lv1.so"
+    CONNECTORS = ("pyodbc", "turbodbc")
+    ENVIRONMENT_NAME = "test"
+    DB_PORT = 8888
+    BUCKETFS_PORT = 6666
+
+
+ODBCINST_INI_TEMPLATE = (
+    cleandoc(
+    """
+    [ODBC]
+    #Trace = yes
+    #TraceFile =~/odbc.trace
+
+    [EXAODBC]
+    #Driver location will be appended in build environment:
+    DRIVER={driver}
+    """)
+    + "\n"
+)
+
+
+def find_session_runner(session, name):
+    for s, _ in session._runner.manifest.list_all_sessions():
+        if name in s.signatures:
+            return s
+    session.error(f"Could not find a nox session by the name {name!r}")
+
+
+def transaction(connection, sql_statements):
```

Comment:
maybe more like this

```suggestion
def run_queries_in_transaction(connection, sql_statements):
```